README.rst

```diff
@@ -4,14 +4,71 @@
 .. image:: https://travis-ci.org/camptocamp/marabunta.svg?branch=master
     :target: https://travis-ci.org/camptocamp/marabunta
 
-Marabunta is a name given to the migration of the legionary ants or to the ants
-themselves. Restless, they eat and digest everything in their way.
+*Marabunta is a name given to the migration of the legionary ants or to the ants
+themselves. Restless, they eat and digest everything in their way.*
 
-This tool aims to run migrations for Odoo versions as efficiently as a
-Marabunta migration.
+Marabunta is used to provide an easy way to provide updates to Odoo which can be created fast and run easily. It also allows to differentiate between different environment to provide for instance demodata.
+
+
+Usage
+=====
+After installing marabunta will be available as a console command. To run properly it requires a migration file () which defines what should be updated/executed) and odoos connection parameters (view options in the options section.
```

Comment:
odoo **'** s connection parameters (**cf** options section **)**.

Reply:
@grindtildeath cf?

Reply:
Trailing parenthesis still missing. (and option is repeated)